extensions/amp-story/1.0/amp-story-request-service.js

```diff
@@ -15,10 +15,12 @@
  */
 
 import {Services} from '../../../src/services';
+import {childElementByTag} from '../../../src/dom';
 import {getAmpdoc} from '../../../src/service';
 import {once} from '../../../src/utils/function';
 import {user} from '../../../src/log';
 
+// TODO(#14591): Replace with 'src' when bookend api v0.1 is deprecated.
```

Comment:
This is in the 1.0 codebase, so you should just be able to replace this now

Reply:
(can you change this before merging?)

Reply:
Oh true. Ok, will do.